ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -221,15 +261,15 @@ const LOCATION_PANEL = {
     title: 'Location',
     name: 'locus',
     inputSize: 5,
-    inputProps: { component: LocusListSelector, format: val => val || {} },
+    inputProps: { component: LocusListSelector, format: val => val || { locusListGuid: '' } },
```

Comment:
why does having `locusListGuid` evaluate to an empty string instead of undefined change things?